pkg/service/manifest/service.go

```diff
@@ -330,7 +330,7 @@ func (s Service) ProcessApplicationSubmission(ctx context.Context, request model
 	return operation.ServiceModel(*storedOp)
 }
 
-// attemptAutomaticIssuance checks if there is an issuance template for the manifest, and if so,
+// attemptAutomaticIssuance checks if there is an issuing template for the manifest, and if so,
```

Comment:
Not sure if you want it issuance or issuing in this file

Reply:
thanks updated